client/packages/config/src/config.ts

```diff
@@ -0,0 +1,21 @@
+import { Capacitor } from '@capacitor/core';
+
+// For production we get URL relative to web app
+const { port, hostname, protocol } = window.location;
+
+const isProductionBuild = process.env['NODE_ENV'] === 'production';
+
+const devServerURL = 'https://demo-open.msupply.org:8000'; // Demo - site URL
```

Comment:
I don't want the URL hard coded in the codebase ( different situation, I think, to have an example URL in a config file 😉  ) - the original intention of the `config.js` file was to allow configuration without needed a different build. if you reinstate something similar then we aren't restricted to a specific URL.

The demo server URL isn't - to me - a 'dev' URL, so the var name seems wrong. Really it would be `demoServerURL = ` but that's way too specific. I would prefer just to fetch from config, perhaps defaulting to `https://localhost:8000` if config is unset

Reply:
> I don't want the URL hard coded in the codebase ( different situation, I think, to have an example URL in a config file 😉 ) - the original intention of the config.js file was to allow configuration without needed a different build. if you reinstate something similar then we aren't restricted to a specific URL.

The idea for this PR was to not even worry about setting config.js in production, since remote-server is 'hosting' front end the api would always be relative to front end window.url. I wanted to reduce configurations required when setting up server. 

> The demo server URL isn't - to me - a 'dev' URL

I thought it was used in dev a lot, since it's in the repo and even has a yarn dev script to run against it `yarn start-remote`, and previously it was the main api url when in dev

Reply:
the demo url is a convenient default and is handy for new devs working on f/e code ( no need to set up mSupply datafile, remote server etc ) so it has been used a fair bit up till now.

it's in a config file which can be updated independently of a build - I see that as different from being in the repo.. more like an example in a config 😉  and the `yarn start-remote` starts the app using the url from the config file, it does not explicitly start the demo server connection. Only if you use the default value.. 
So the script is `remote` and not `demo`..